snippets/Adder/Adder.ml

```diff
@@ -0,0 +1,111 @@
+module CB_C = struct
```

Comment:
where should this be? perhaps we can do examples/protocols for all the "standalone" examples (what we have now), examples/implementations for these kind of examples? or put this implemented example as documentation

Reply:
I put it in `snippets` because we choose `examples` to be scribble examples, I think it's worthwhile writing a tutorial - but for later